### PR TITLE
Fix flashcard multiline description

### PR DESCRIPTION
**Pull request by JJthere**
## Description

This PR fixes flashcard multiline descriptions, by changing the InputField to a TextArea.

## Resolved Issue

Resolves part of #120 (card side description is limited to one line)

### CODE REVIEW

File: frontend/components/form/TextArea.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ */
+export const TextArea = forwardRef(
+	(
+		{ name, className = "", ...props }: TextAreaProps,
+		ref: Ref<HTMLTextAreaElement>
+	) => (
+		<div className={`flex h-full w-full flex-col ${className}`}>
+			<textarea
+				name={name}
+				className={` h-full w-full resize-none rounded-md border-eggshell bg-eggshell font-medium text-kiokuLightBlue outline-none`}
```
Comment (Julian702) [inline frontend/components/form/TextArea.tsx:45]: Is there a reason to use a border in the same color as the background which is not even visible?
Maybe also think about making the background transparent to use the component more universally.
```suggestion
				className={` h-full w-full resize-none rounded-md bg-transparent font-medium text-kiokuLightBlue outline-none`}
```

**Review by Julian702 — CHANGES_REQUESTED** [review]
Good job, found just one small nit. 👍

File: frontend/components/form/TextArea.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ */
+export const TextArea = forwardRef(
+	(
+		{ name, className = "", ...props }: TextAreaProps,
+		ref: Ref<HTMLTextAreaElement>
+	) => (
+		<div className={`flex h-full w-full flex-col ${className}`}>
+			<textarea
+				name={name}
+				className={` h-full w-full resize-none rounded-md bg-transparent font-medium text-kiokuLightBlue outline-none`}
```
Comment (darmiel) [inline frontend/components/form/TextArea.tsx:45]:
```suggestion
				className="h-full w-full resize-none rounded-md bg-transparent font-medium text-kiokuLightBlue outline-none"
```

File: frontend/components/form/TextArea.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	/**
+	 * Is the TextArea read only?
+	 */
+	readOnly?: boolean;
+	/**
+	 * Additional classes
+	 */
+	className?: string;
+	/**
+	 * optional change handler
```
Comment (darmiel) [inline frontend/components/form/TextArea.tsx:29]:
```suggestion
	 * Change handler
```

File: frontend/components/form/TextArea.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import React, { ChangeEventHandler, Ref, forwardRef } from "react";
+
+interface TextAreaProps {
+	/**
+	 * unique identifier
```
Comment (darmiel) [inline frontend/components/form/TextArea.tsx:5]:
```suggestion
	 * Unique identifier
```

**Review by darmiel — COMMENTED** [review]
Good job, found just three small nits. 👍

**Review by memeToasty — APPROVED** [review]

File: frontend/components/form/TextArea.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+	readOnly?: boolean;
+	/**
+	 * Additional classes
+	 */
+	className?: string;
+	/**
+	 * Change handler
+	 */
+	onChange?: ChangeEventHandler<HTMLTextAreaElement>;
+}
```
Comment (darmiel) [inline frontend/components/form/TextArea.tsx:32]: any reason to use these predefined props and not just allow all HTML Text Area props?

**Review by darmiel — APPROVED** [review]

**Review by TomRomeo — APPROVED** [review]
💖

File: frontend/components/form/TextArea.tsx (hunk 5 above)
Comment (JJthere) [reply frontend/components/form/TextArea.tsx:32]: No,
we will need to address this problem in another pr, because of its reoccurrence in other files.

**Review by 4KevR — APPROVED** [review]

**Pull request merged**